.github/workflows/test.yml

```diff
@@ -10,165 +10,151 @@ on:
     - cron:  '0 * * * *'
 
 jobs:
-  test:
+  test_environment:
+    name: 'Check development tools'
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-          os: [macOS-latest, ubuntu-18.04, windows-latest]
+          os: [macOS-latest, windows-latest]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - uses: actions/setup-node@v1
       with:
         node-version: '12.x'
-    - run: npm ci
-    - run: npm run build
-    - run: npm test
-
+    - run: ./.github/workflows/build-and-test.sh
```

Comment:
Do we need to use `github.workspace`? This should be fine IMO tho.

Reply:
Done!